monitoring/uss_qualifier/suites/faa/uft/design/README.md

```diff
@@ -0,0 +1,79 @@
+# UFT Message Signing tests for SCD using Uss_Qualifier
+
+UTM Field Test (UFT) has a requirement for USSes to sign their message requests
+and responses in the SCD flow using IATF certificates provided by FAA.
+
+This test suite helps to test the message signing by extending [suites.astm.utm.f3548_21](../../../astm/utm/f3548_21.yaml)
+with scenarios that trigger reporting of message signing in the interactions with mock_uss.
```

Comment:
Is it expected that all USSs capture the report and make it available to the FAA or is it going to be generated only by a specific component of the system ?
Is there a plan to run on a distributed environment or only locally by each USS organization ?

Reply:
At the moment the plan is to run the components locally by USSes for generating the reports - both uss_qualifier and message_signing. I am not sure if any USS has run the uss_qualifier tests in a distributed environment in the past. Probably, Ben would know about it. Eventually, I think these tests should be run in a distributed environment, ready for any USS in production to come and get tested.